spec/index.html

```diff
@@ -851,7 +851,17 @@ <h4 style="padding-bottom:0; margin-bottom:0;"><span>Collections</span></h4>
 
 <section id="privacy-considerations" class="appendix informative">
   <h2>Privacy Considerations</h2>
-  <p class="ednote">TODO</p>  
+  <p>TriG is used to express arbitrary application data,
```

Comment:
As with the others...
```suggestion
  <p>The TriG format is used to express arbitrary application data,
```